docs/documentation/basics/batching-simulations.md

```diff
@@ -114,6 +111,8 @@ The simulation runs for each set of Hamiltonians, jump operators and initial sta
 
     For example for `dq.sesolve()` with `H` of shape _(2, 3, n, n)_, `psi0` can be of shape: _(n, 1)_, _(3, n, 1)_, _(2, 1, n, 1)_, _(2, 3, n, 1)_, _(..., 2, 3, n, 1)_, etc. By playing with the arguments shape, you have complete freedom over the simulation you want to run.
 
+??? Note "What is broadcasting?"
+    JAX and NumPy broadcasting semantics are very powerful and allow you to write concise and efficient code. For more information, see the [NumPy documentation on broadcasting](https://numpy.org/doc/stable/user/basics.broadcasting.html).
```

Comment:
I think it was better before. For people who don't know about broadcasting, they could check what it means. Now, they might read to the end of the section without understanding it.